src/App.js

```diff
@@ -1,26 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
```

Comment:
- [ ] The `App.js` component should be class-based as per the project's requirements even though the functional component is very correct. Please change this functional component to a class component. :+1:
- [ ] You also need to import `React, {Component}` when you convert the functional component to class component extended from `React Component`.